Framework/PythonInterface/test/python/plugins/algorithms/SaveYDATest.py

```diff
@@ -218,8 +218,9 @@ def _create_workspace(self, ws_2D=True, sample=True, xAx=True, yAxSpec=True, yAx
         if not yAxMt and not yAxSpec:
             ws = CreateWorkspace(DataX=self.data_x, DataY=self.data_y, DataE=np.sqrt(self.data_y), NSpec=1, UnitX="DeltaE")
             LoadInstrument(ws, True, InstrumentName="TOFTOF")
-            ConvertSpectrumAxis(InputWorkspace=ws, OutputWorkspace=ws, Target="theta", EMode="Direct")
-            return ws
+            ConvertSpectrumAxis(InputWorkspace=ws, OutputWorkspace="ws2", Target="theta", EMode="Direct")
+            ws2 = mtd["ws2"]
+            return ws2
```

Comment:
```suggestion
            ouput_ws = ConvertSpectrumAxis(InputWorkspace=ws, Target="theta", EMode="Direct")
            return ouput_ws 
```

Reply:
ConvertSpectrumAxis will return the output workspace if we do not provide a string to the `OutputWorkspace` property

Reply:
Okay 👍